packages/client/katello-host-tools/katello-host-tools.spec

```diff
@@ -81,6 +81,10 @@ BuildRequires: python2-devel
 %endif
 %endif
 
+%if 0%{?rhel} == 6
+Requires: python-setuptools
```

Comment:
Wouldn't this also be an issue on EL7? Or are the scripts not used there?

Reply:
I need to test this. My assumption was this would've also been reported for EL7, but then again that katello-enabled-repos-upload script isn't the default any more